src/states/finitemps.jl

```diff
@@ -129,13 +133,44 @@ function Base.getproperty(ψ::FiniteMPS, prop::Symbol)
         return ARView(ψ)
     elseif prop == :AC
         return ACView(ψ)
-    elseif prop == :CR
-        return CRView(ψ)
+    elseif prop == :C
+        return CView(ψ)
+    elseif prop == :center
+        return _gaugecenter(ψ)
     else
         return getfield(ψ, prop)
     end
 end
```

Comment:
Probably it also makes sense to add `Base.propertynames` to reflect our implementation:
https://docs.julialang.org/en/v1/base/base/#Base.propertynames

Reply:
One other thing is that it might be reasonable to not actually pass on the `ACs` etc to `getfield`, and instead require that these are called manually through `getfield`, in order to reflect that they are considered internal

Reply:
`Base.propertynames` is now implemented.
Right now the docstring for `FiniteMPS` only shows the properties, not the fields. If people want to access the underlying `.ACs`, they know they exist and therefore probably know what they are doing.

To be honest, only being able to access \`.ACs\` via `getfield` will make for lots of interal changes to the code and will make it less readable imo.